onecodex/notebooks/exporters.py

```diff
@@ -91,8 +91,9 @@ def from_notebook_node(self, nb, resources=None, **kw):
         elif "metadata" not in resources:
             resources["metadata"] = {}
 
+        do_not_insert_date = os.environ.get("ONE_CODEX_DO_NOT_INSERT_DATE") == "True"
```

Comment:
Double negative could be confusing - would it be clearer to just use INSERT_DATE instead of DO_NOT_INSERT_DATE?

Reply:
Good point. Fixed.